src/app/authentication/components/form-login/form-login.component.html

```diff
@@ -1,11 +1,25 @@
-<form class="form-login">
+<div *ngIf="errorMessage != ''">
+  <div class="alert alert-dark">
```

Comment:
This could be moved to its own component too later on

Reply:
Thanks @olivierobert for point me out, I created [an issue](https://github.com/carryall/survey-challenge/issues/35) for moving the alert to component.